src/PrestaShopBundle/Twig/HookExtension.php

```diff
@@ -165,16 +167,43 @@ public function renderHook($hookName, array $hookParameters = [])
             ->outputContent();
     }
 
+    /**
+     * Return the concatenated content of a renderHooksArray response
+     *
+     * @param array $hooksArray the array returned by the renderHooksArray function
+     *
+     * @return string
+     */
+    public function hooksArrayContent($hooksArray)
+    {
+        $content = '';
+        if (!is_array($hooksArray)) {
+            return $content;
+        }
+
+        foreach ($hooksArray as $hook) {
+            if (isset($hook['content'])) {
+                $content .= $hook['content'];
+            }
+        }
+
+        return $content;
+    }
+
     /**
      * Count how many listeners will respond to the hook name.
      * Does not trigger the hook, so maybe some listeners could not add a response to the result.
      *
+     * @deprecated since 1.7.7.0
+     *
      * @param string $hookName
      *
      * @return number the listeners count that will respond to the hook name
      */
     public function hookCount($hookName)
     {
+        @trigger_error('The ' . __METHOD__ . ' method is deprecated since version 1.7.7.0.', E_USER_DEPRECATED);
```

Comment:
```suggestion
        @trigger_error('The ' . __METHOD__ . ' method is deprecated since version 1.7.7.0. Please use hooksArrayContent.', E_USER_DEPRECATED);
```

Reply:
I thought about it but `hooksArrayContent` is not aim to be a replacement of the `hookCount` function. In our case we can use `hooksArrayContent` for the same result but it's probably not always the case. WDYT?